Source/ImagePickerController.swift

```diff
@@ -340,20 +312,7 @@ extension ImagePickerController: ImageGalleryPanGestureDelegate {
           }
       })
     } else if velocity.y > 100 || galleryView.frame.size.height - galleryView.topSeparator.frame.height < 100 {
-      UIView.animateWithDuration(0.2, animations: { [unowned self] in
```

Comment:
I like to keep everything concise, so you can understand `expected` behavior just by reading code. Then, you can go to implementation details of particular method and fix it there, without being distracted by all the surrounding if/else/etc.
Good job!